src/layouts/article.ejs

```diff
@@ -2,15 +2,17 @@
 <html lang="zxx" class="no-js">
 	<head>
 		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-		<link rel="shortcut icon" href="/img/small_logo.png">
+		<link rel="shortcut icon" href="/img/logo_fb.png">
 		<meta name="author" content="betaflight">
 		<meta name="description" content="">
 		<meta name="keywords" content="">
 		<meta charset="UTF-8">
-		<meta property="og:image" content="https://betaflight.com/img/small_logo.png">
+		<meta property="og:image" content="https://betaflight.com/img/logo_fb.png">
 		<meta property="og:description" content="<%= page.description %>">
 		<meta property="og:title" content="betaflight.com - <%= page.title %>">
-		
+		<meta property="og:url" content="https://betaflight.com/">
+		<meta property="og:type" content="website">
```

Comment:
The URL here should be the article url, however it’s not available at this point in nanogen. I’m submitting a PR there to fix that. So this’ll do in the interim.

Reply:
That is for `og:url`? Not entirely sure about that, according to https://developers.facebook.com/docs/sharing/webmasters/ it should be the 'canonical URL', and the 'likes and shares are aggreagated' bit (to me) sounds like using the same URL for all articles is probably a better approach. But I am not an expert in Facebook optimisation.